src/http_link.c

```diff
@@ -68,7 +70,8 @@ void http_read_cb_override(uv_link_t *link, ssize_t nread, const uv_buf_t *buf)
         int status_line_len;
         for (status_line_len = 0; status_line_len < nread && buf->base[status_line_len] != '\r'; ++status_line_len)
           ;
```

Comment:
what is that semicolon doing here? :)

Reply:
Vs code did it ;) This for loop doesn't have a body, but I should replace this for with function anyway.

Reply:
okay, then do this.